### DISCUSSION TITLE
One To Many mapping with Copyto

### DISCUSSION THREAD

**Discussion by rajkumar0rk** [atlasmap/atlasmap#4612] [Q&A]
Hi Team,
I am trying to map OneToMany

Source JSON
{
    "profile": {
        "firstName": "jone",
        "lastName": "alex"
    }
}

Target JSON

{
    "user": {
        "profile": [
            {
                "name": "jone"
            },
            {
                "name": "alex"
            }
        ]
    }    
}

When i trying map firstname to name with action(Copyto with index 1) and lastname to name with action(Copyto with index 2)
![Screenshot 2024-04-02 194043](https://github.com/atlasmap/atlasmap/assets/94229514/2860cf5f-950e-4d7b-8f28-2131e2a58e58)
![Screenshot 2024-04-02 194222](https://github.com/atlasmap/atlasmap/assets/94229514/00265855-85c1-4f93-a8f5-57f7595a78f3)

I am getting below output. firstName is missing

{"user":{"profile":[{"name":"alex"}]}}

Guide me if this procedure is correct or any other suggestion

**Comment by arockiaraj1994** [comment]
Try to use repeat rather than coptyto. This is workaround only.
